sympy/matrices/tests/test_matrices.py

```diff
@@ -597,33 +597,6 @@ def test_issue_17247_expression_blowup_30():
         [-15091965363354518272/21217636514687010905 + 16890163109293858304*I/21217636514687010905],
         [                                                          -11328/952745 + 87616*I/952745]]'''))
 
-# This test is commented out because without dotprodsimp this calculation hangs.
-# @XFAIL # dotprodsimp is not on by default in this function
-# def test_issue_17247_expression_blowup_31():
-#     M = Matrix([
-#         [x + 1, 1 - x,     0,     0],
-#         [1 - x, x + 1,     0, x + 1],
-#         [    0, 1 - x, x + 1,     0],
-#         [    0,     0,     0, x + 1]])
-#     assert M.LDLsolve(ones(4, 1)) == Matrix([
-#         [(x + 1)/(4*x)],
-#         [(x - 1)/(4*x)],
-#         [(x + 1)/(4*x)],
-#         [    1/(x + 1)]])
-
-@XFAIL # dotprodsimp is not on by default in this function
-def test_issue_17247_expression_blowup_32():
```

Comment:
You can ignore the tests like `expression_blowup`.

Reply:
Thank you very much for the suggestion.